pkgs/development/python-modules/python-lorem/default.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  python3Packages,
+}:
+buildPythonPackage rec {
+  pname = "python-lorem";
+  version = "1.3.0.post3";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "python_lorem";
+    hash = "sha256-Vw1TKheXg+AkhksnmWUfdIo+Jt7X7m1pS2f0Kfe8pv0=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    python3Packages.pytestCheckHook
+  ];
```

```suggestion
  build-system = [
    setuptools
  ];

  nativeCheckInputs = [
    pytestCheckHook
  ];
```

function inputs of `python3Packages.callPackage` (which is the case for all in-tree `python3Packages` attributes) are the expansion of `python3Packages` on top of top-level packages.